securedrop/journalist_templates/base.html

```diff
@@ -20,6 +20,15 @@
   <body>
 
     {% if g.user %}
+
+        {% if g.show_xenial_warning %}
+    <div class="xenial-upgrade critical">
+        <img class="critical-skull" src="{{ url_for('static', filename='i/font-awesome/skull-crossbones-white.png') }}" 
+             width="14px" height="16px">
+        {{ gettext ('<strong>Critical Security:</strong> The operating system used by your SecureDrop servers has reached its end-of-life. A manual update is urgently required to remain safe - <a href="//securedrop.org/xenial-upgrade" rel="noreferrer">Learn More</a>') }}
```

Comment:
The link to the article posted just now https://securedrop.org/news/advisory-preparing-server-upgrade-ubuntu-1404-1604/ . Perhaps we should update it as part of this PR?

Reply:
I'd suggest linking to https://docs.securedrop.org/en/latest/upgrade/xenial_prep.html for now - that has the same intro, and will ultimately be cross-linked to the full Xenial upgrade guide in the docs.

Reply:
Since we don't have final docs right now, and neither of those really work as entrypoints to the process, this is why I suggested in #4027 adding a redirect from the generic securedrop.org URL in the message to whatever we want people to be reading.

That seems more flexible to me, though it does mean there needs to be a task to add said redirect on securedrop.org as part of the 0.12.0 release prep. https://docs.securedrop.org/en/latest/upgrade/xenial_prep.html  does make the most sense as a first candidate for the destination of the redirect, IMO.

Reply:
@zenmonkeykstop I'd initially missed your recommendation in #4027 for a redirect on SDO to point to wherever the final docs for the Xenial upgrade land. You're right to flag a bit of infra involvement to place that redirect, but I agree it provides us maximum flexibility. Will get that ball rolling targeting https://docs.securedrop.org/en/latest/upgrade/xenial_prep.html as both you and @eloquence suggest.

Reply:
+1 to @eloquence's tweak on using the "stable" URL.

Reply:
Just following up here, the https://securedrop.org/xenial-upgrade redirect is now live